test_runner/regress/test_wal_acceptor.py

```diff
@@ -2083,8 +2083,13 @@ def remember_lsn():
 
     endpoint.safe_psql("create table t(key int, value text)")
 
-    timeline_status = env.safekeepers[0].http_client().timeline_status(tenant_id, timeline_id)
-    timeline_start_lsn = timeline_status.timeline_start_lsn
+    # Note: currently timelines on sks are created by compute and commit of
```

Comment:
> ... timelines on sks are created by compute ...

What exactly does that mean? The compute doesn't make an explicit "create timeline" call to the safekeepers

Reply:
It is created on first connection from walproposer (which eventually will be fixed).